src/domain/usecases/data-entry/egasp/CustomValidationForEGASP.ts

```diff
@@ -64,11 +66,11 @@ export class CustomValidationForEGASP {
     private checkPeriod(events: Event[], period: string): ConsistencyError[] {
         const errors = _(
             events.map(event => {
-                const eventDate = new Date(event.eventDate);
-                if (eventDate.getFullYear().toString() !== period) {
+                const eventDate = new Date(event.occurredAt);
+                if (eventDate < firstDayOfQuarter(period) || eventDate > lastDayOfQuarter(period)) {
```

Comment:
@davidbgomes I think the changes to this file src/domain/usecases/data-entry/egasp/CustomValidationForEGASP.ts are accidentally added. EGASP was initially a quarterly period module and then was changed to a Yearly period module. The changes in this file are reverting the yearly period changes. While testing EGASP flow, the upload crashes due to this file. Could be a merge issue? Can you check this out?